index.html

```diff
@@ -173,6 +173,20 @@ <h2>Recipes</h2>
                     </div>
                 </div>
             </div>
+            <div class="col-md-4 col-sm-6 hero-feature">
+                <div class="thumbnail">
+                    <a href="https://ghcguide.haskell.jp">
+                        <h2>GHC User's Guide</h2>
+                        <img src="img/study-rabbit.svg" alt="勉強する兎" style="margin-left: 30%; margin-right: 30%;">
+                    </a>
+                    <div class="caption">
+                        <p>代表的なHaskellコンパイラであるGHCのドキュメントです。オプション、言語拡張、インタプリタなどに関して困ったときは確認してみましょう！</p>
```

Comment:
翻訳中であることを明記した方が混乱が少ないかと思います。
あと、翻訳したい人への導線にもなるので、リポジトリーへのリンクも張りましょう（先ほどそのためにフォークしました）。

```suggestion
                        <p>代表的なHaskellコンパイラであるGHCのドキュメントです。オプション、言語拡張、インタプリタなどに関して困ったときは確認してみましょう！<br />
                        <strong>現在翻訳中です！</strong><a href="https://github.com/haskell-jp/ghc_users_guide_ja">こちらのリポジトリー</a>にPull requestを送って貢献することもできます！
                        </p>
```

Reply:
PR ページにリンクしてみましたが、現状 PR が空なので、普通にリポジトリのトップページの方がいいですかね？

Reply:
リポジトリーのトップでいいと思います！

Reply:
トップに変更しました。